### ISSUE TITLE
Avalonia Foreground Color Broken on macOS

### ISSUE THREAD

**Issue by hdocsek** [davidxuang/FluentIcons#7]
I discovered that with the latest version of FluentIcons.Avalonia on `macOS (Sonoma 14.4.1)` the Foreground color is not applied. However, this works on `Win 11` as well as on `Linux Ubuntu 22.04`.

The behavior was introduced in `FluentIcons.Avalonia v1.1.230`, but it was still working in version `FluentIcons.Avalonia v1.1.228`.
I could not figure out the root cause, but I noticed that the fonts have been updated between these versions. Maybe this is related.

**Behavior/Issue:**
SymbolIcon Foreground color is not applied on `macOS`.

**Steps to reproduce:**

*Project File:*
```
<Project Sdk="Microsoft.NET.Sdk">
    ...
    <ItemGroup>
        <!--  Broken starting with 1.1.230 -->
        <PackageReference Include="FluentIcons.Avalonia" Version="1.1.233" />
        <!--  Working -->
        <!-- <PackageReference Include="FluentIcons.Avalonia" Version="1.1.228" /> -->
    </ItemGroup>
    ....
</Project>
```

*MainWindow.axaml*
```
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:ic="clr-namespace:FluentIcons.Avalonia;assembly=FluentIcons.Avalonia"
        mc:Ignorable="d" d:DesignWidth="800" d:DesignHeight="450"
        x:Class="SymbolForegroundIssue.MainWindow"
        Title="SymbolForegroundIssue">
    
    <ic:SymbolIcon Symbol="Home" IsFilled="True" Foreground="Red" />
    
</Window>
```

**Expected Outcome:**

SymbolIcon Foreground color should be Red on `macOS`.

**Comment by davidxuang** [comment]
I suspect the reason is that Avalonia or SkiaSharp can't handle OpenType [COLR](https://learn.microsoft.com/en-us/typography/opentype/spec/colr) table on macOS. Starting with version 1.1.230, COLR table is added to the font to render color flags properly, but it didn't cover, and shouldn't affect other characters. I suppose this should be forwarded to upstream.

**Comment by davidxuang** [comment]
@hdocsek I've uploaded an experimental version `1.1.234-exp` for Avalonia, with COLR table dropped from TTF files. If my guess was correct, this version should work on macOS.
I couldn't find an environmet to reproduce the issue shortly, so could you test this version to validate the theory?

**Comment by hdocsek** [comment]
@davidxuang Seems like you were spot on. Version `1.1.234-exp` works correctly on macOS.

**Comment by davidxuang** [comment]
@hdocsek Could you also attach a screenshot for both `1.1.234` and `1.1.234-exp`? Please include both a regular icon and a color icon, like:

```xaml
<StackPanel>
    <ic:SymbolIcon Symbol="Home" IsFilled="True" />
    <ic:SymbolIcon Symbol="Home" IsFilled="True" Foreground="Red" />
    <ic:SymbolIcon Symbol="FlagPride" IsFilled="True" />
    <ic:SymbolIcon Symbol="FlagPride" IsFilled="True" Foreground="Red" />
</StackPanel>
```

**Comment by hdocsek** [comment]
@davidxuang Here are the screenshots for both versions:
<img width="403" alt="FluentIcons Avalonia 1 1 234-labeled" src="https://github.com/davidxuang/FluentIcons/assets/2975344/e2e94c6a-f6f7-4fa3-8df6-2bbf6635ab53">
<img width="405" alt="FluentIcons Avalonia 1 1 234-exp-lableled" src="https://github.com/davidxuang/FluentIcons/assets/2975344/ef4b7ebe-26e5-4953-b2a1-01e9fb170e99">

**Comment by davidxuang** [comment]
@hdocsek Can you test if this issue also applies to SkiaSharp 3 preview? It seems to me Avalonia 11.1 beta series work with SkiaSharp 3 preview.

```csproj
    <PackageReference Include="Avalonia" Version="11.1.0-beta1" />
    <PackageReference Include="SkiaSharp" Version="3.0.0-preview.2.1" />
```

**Comment by hdocsek** [comment]
@davidxuang Referencing SkiaSharp `3.0.0-preview.0.132` seems to fix the issue no matter if Avalonia `11.0.10` or `11.1.0-beta1` is used.

<img width="512" alt="FluentIcons Avalonia 1 1 234_WithSkiaSharp-labeled" src="https://github.com/davidxuang/FluentIcons/assets/2975344/a8bddb6a-1b77-441f-b5e8-4f07b704420e">
